front/pages/api/v1/w/[wId]/members/validate.ts

```diff
@@ -0,0 +1,85 @@
+import type { WithAPIErrorResponse } from "@dust-tt/types";
+import { isLeft } from "fp-ts/lib/Either";
+import * as t from "io-ts";
+import * as reporter from "io-ts-reporters";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { withPublicAPIAuthentication } from "@app/lib/api/wrappers";
+import type { Authenticator } from "@app/lib/auth";
+import { MembershipResource } from "@app/lib/resources/membership_resource";
+import { UserResource } from "@app/lib/resources/user_resource";
+import { renderLightWorkspaceType } from "@app/lib/workspace";
+import { apiError } from "@app/logger/withlogging";
+
+export type ValidateMemberResponseBody = {
+  valid: boolean;
+};
+
+/**
+ * @ignoreswagger
+ * Validates an email corresponds to an active member in a specific workspace. For Dust managed apps only - undocumented.
+ */
+
+export const validateEmailSchema = t.type({
```

Comment:
Can you use a name aligned with other endpoints :pray:

Reply:
-> PostValidateMemberRequestBodySchema